_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp

```diff
@@ -2692,7 +2692,10 @@ void SetDefaults(
         if (!par.MaxKbps)
             par.MaxKbps = par.TargetKbps;
         if (!par.BufferSizeInKB)
+        {
             par.BufferSizeInKB = Min(maxBuf, par.MaxKbps / 4); //2 sec: the same as H264
+            par.BufferSizeInKB = Max(par.BufferSizeInKB, par.InitialDelayInKB);
```

Comment:
please, use std::min / max

Reply:
Why?

Reply:
Because there is std:min/max - a tool coming from standard library. Self-defined Min/Max to be completely removed from mediasdk (not in this commit). Basic rule: don't define something if it already exists.

Reply:
I didn't define Min, Max in my commit. If you decide to remove it, it must be replaced by one commit in all code. It is better to have code in one style.

Reply:
And you suggest to merge in a change which uses Min in one place and MFX_MIN in another place? Code review is a reason to once again check the code, find issues and decided what we will do with them. Multiple different min/max primitives is definitively a problem. I believe that we need to make sure we use single standard approach right away in this commit, i.e. std::min/max and plan a task to change everything else.

Please, use std::min/max everywhere within your change.

Reply:
Min, Max is used in hevc encoder
MFX_MIN, MFX_MAX - in h264 encoder

for replacement it single commit is needed. There are a lot of calls.

Reply:
I don't think so. We can and should write correctly those places we change.